repo2docker/buildpacks/pipfile/__init__.py

```diff
@@ -0,0 +1,133 @@
+"""Generates Dockerfiles based on an input matrix based on Python."""
```

Comment:
Let's update this to say something about Pipenv

Reply:
Resolved by 2189777